hyperspy/signal.py

```diff
@@ -4051,6 +4066,10 @@ def min(self, axis, out=None):
         axis : {int | string}
            The axis can be specified using the index of the axis in
            `axes_manager` or the axis name.
+        out : {Signal, None}
```

Comment:
Sure, I was thinking about this myself, but I figured I'd see what you though :)